platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -262,31 +266,57 @@ public void run() {
         });
     }
 
-    static MapController getInstance(GLSurfaceView view) {
-        return new MapController(view);
-    }
 
     /**
-     * Load a new scene file
+     * Load a new scene file.
+     * Use {@link #setSceneLoadListener(SceneLoadListener)} for notification when the new scene is
+     * ready.
      * @param path Location of the YAML scene file within the application assets
+     * @return Scene ID
```

Comment:
"An identifier for the scene being loaded, the same value will be passed to `SceneLoadListener.onSceneReady` when loading is complete." Does that sound right?